### DISCUSSION TITLE
Changing the color of the header section!!

### DISCUSSION THREAD

**Discussion by RadheshRagunath** [squidfunk/mkdocs-material#2581] [Q&A] [answered]
I would like to know if it's possible to change the color of the header section.
![image](https://user-images.githubusercontent.com/81693941/115237550-515a9c80-a13a-11eb-91ca-5df7e6d7c75d.png)
![image](https://user-images.githubusercontent.com/81693941/115237676-76e7a600-a13a-11eb-92b6-8c646200f2a0.png)
As you can see from the above images, "Changing the color" font color is white for majority of the available colors but it's black for the colors "lime, orange, amber and yellow". Is it possible to change the color to "white" in these four cases?

Thank you in advance for your suggestions...

**Comment by martinbira** [comment, marked as answer]
Have you checked https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#customization ?

**Reply by RadheshRagunath** [reply]
Yes, I have checked, but don't find any help there for the above case.

**Reply by martinbira** [reply]
I think that it has the answer. If you set your colour scheme to `orange` for example, change the `--md-primary-bg-color` as such with additional stylesheet: 

```
[data-md-color-primary="orange"] {
	--md-primary-bg-color: #fff;
}
```

**Reply by RadheshRagunath** [reply]
Thank you very much @martinbira . 
It works fine.
